includes/User.php

```diff
@@ -4903,19 +4903,15 @@ private function loadAttributes() {
 	}
 
 	private function compareAttributeValueFromService( $attributeName, $attributeValue ) {
-		if ( !array_key_exists( $attributeName, $this->mOptions ) ) {
```

Comment:
Now that we're iterating over `$wgPublicAttributes` we don't need to check if any are missing since we'll be comparing all possible values the attribute service could return. See: https://github.com/Wikia/app/commit/99c6426e8aa35640270752283d4a76d78ccafa2a